extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/conf/ConfigurationService.java

```diff
@@ -391,5 +392,21 @@ public MemberAttributeType getMemberAttributeType()
             MemberAttributeType.DN
         );
     }
+    
+    /**
+     * Return the default configuration that determines how Guacamole Client
+     * connects to guacd in order to establish connections to remote servers.
+     * 
+     * @return
+     *     The default configuration used by Guacamole Client for connecting
+     *     to guacd.
+     * 
+     * @throws GuacamoleException 
+     *     If guacamole.properties cannot be parsed.
+     */
+    public GuacamoleProxyConfiguration getDefaultGuacamoleProxyConfiguration()
+            throws GuacamoleException {
+        return environment.getDefaultGuacamoleProxyConfiguration();
+    }
```

Comment:
This may not be needed depending on whether we can simply rely on `SimpleConnection` to pull this for us.

Reply:
Removed.